FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -28,11 +24,13 @@
  */
 #define FBSnapshotVerifyViewWithOptions(view__, identifier__, suffixes__) \
 { \
+NSString *envReferenceImageDirectory = [NSProcessInfo processInfo].environment[@"FB_REFERENCE_IMAGE_DIR"]; \
```

Comment:
I mean [does support](https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/BuildingCocoaApps/InteractingWithCAPIs.html) preprocessor conditionals (under the Build Configurations heading).

So we can have backwards compatibility and not impact build scripts. I haven't hit the unexpanded macro problem - is that when the dependency chain is: app under test -> some lib -> FBSnapshotTestCase?